src/components/BlockingViews/FullPageOfflineBlockingView.js

```diff
@@ -5,6 +5,7 @@ import networkPropTypes from '@components/networkPropTypes';
 import {withNetwork} from '@components/OnyxProvider';
 import withLocalize, {withLocalizePropTypes} from '@components/withLocalize';
 import compose from '@libs/compose';
+import themeColors from '@styles/themes/default';
```

Comment:
`themeColors` should not be used directly. Use the `useTheme` hook instead please